source/documentation/managing_apps/scaling.md

```diff
@@ -12,10 +12,12 @@ from the command line.
 
 If you are anticipating a spike in demand for a service hosted on GOV.UK PaaS, please contact us well in advance at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
 
-###Increasing instances
+### Increasing instances
 
-You can change the number of instances of your app running in parallel.
-Incoming requests are automatically load-balanced across all instances.
+You can change the number of instances of your app running in parallel. Running more than one app instance:
+
+- allows your app to handle increased traffic and demand as incoming requests are automatically load-balanced across all instances
+- helps maintain high availability and decreases the likelihood that the failure of a cell will take down your app
```

Comment:
> failure of a cell

Do we mention the concept of "cells" anywhere else? ... this might be a confusing new term.

Maybe:

> helps maintain high availability by decreasing the likelihood that the failure of a single component could take down your app

Reply:
agree, we don't mention cell anywhere else in the docs so component is an acceptable generic term